api/descriptions/feeds/malware_get.md

```diff
@@ -1,11 +1,37 @@
-Retrieves the custom list of malware signatures.
+Retrieves the customized list of malware MD5 signatures of malicious executables.
```

Comment:
@KICKdesigns Small edit:

`Retrieves the customized list of MD5 signatures of malicious executables.`

(i.e. `malware MD5 signatures` -> `MD5 signatures`